config/development.json

```diff
@@ -71,7 +71,6 @@
 		"gutenboarding/plans-grid": true,
 		"help": true,
 		"help/courses": true,
-		"home/layout-dev": true,
```

Comment:
I'm pretty neutral to whether this is on in dev or not. If this lands, folks will want to add `calypso.localhost:3000/home/siteSlug?flags=home/layout-dev` to see any in progress home cards.

Reply:
😬 Didn't mean to remove that one--a bit too zealous with the search and replace. I'll add this back.